src/OnboardingSPA/steps/SiteGen/SiteDetails/index.js

```diff
@@ -65,9 +65,10 @@ const SiteGenSiteDetails = () => {
 	}, [] );
 
 	useEffect( () => {
+		// When something new is added then only change in store.
 		if (
 			customerInput !== undefined &&
-			customerInput !== '' &&
+			customerInput?.trim() !== '' &&
```

Comment:
Is it safe to remove the `?` since we already checked for `undefined`? Also, let's trim only once and add the empty string and not matching existing prompt checks as part of a separate condition?

Reply:
Sure.

Reply:
Added changes, Please check, Thanks.